views/season.pug

```diff
@@ -0,0 +1,17 @@
+html
+  include partials/head
+  body
+    h1= showdata.title
+    h2.seasonheader= `Season ${season.number}`
+    table
+      tr(colspan=3)
+        th.col_num No.
+        th.col_title Title
+        th.col_director Director
```

Comment:
Weird to see snakecase here when your other classes don't use it. Be consistent.